### PR TITLE
Add el9 to install script

### PR DESCRIPTION
**Pull request by galal-hussein**
#### Proposed Changes ####
Add el9 to the install script

#### Linked Issues ####

- https://github.com/rancher/rke2-selinux/issues/36
- https://github.com/rancher/rke2/issues/4285

### CODE REVIEW

**Review by briandowns — APPROVED** [review]

File: install.sh [hunk 1]
```diff
@@ -487,7 +487,7 @@ do_install_rpm() {
                     :
                     ;;
                 9) # We are currently using EL8 packages for EL9 as well
```
Comment (brandond) [inline install.sh:489]:
```suggestion
                9)
```

**Review by brandond — CHANGES_REQUESTED** [review]

File: install.sh (hunk 1 above)
Comment (brandond) [reply install.sh:489]: Actually we don't need this at all. Just take out the whole `9)` case and add 9 to the `7|8)` case up above.

**Comment by rancher-max** [comment]
https://github.com/rancher/rke2/issues/4285

File: install.sh [hunk 2]
```diff
@@ -564,6 +571,16 @@ EOF
     fi
 }
 
+check_available_upgrades() {
+    set +e
+    available_upgrades=$(yum -q --refresh list $1 --upgrades | tail -n 1 | awk '{print $2}')
```
Comment (brandond) [inline install.sh:576]: Don't we need to use `${rpm_installer}` for this, not `yum`?

**Pull request merged**